### DISCUSSION TITLE
Filter chart data by foreign key

### DISCUSSION THREAD

**Discussion by thaikoh** [apache/superset#26812] [Q&A / Help] [answered]
I have a question about chart data filtering. 
For example I have a book sales table `sales: date, book, sum, author_id, store_id`; and two tables `author: id, name`; and `store: id, name`. 
The goal is to see total sales for all books and make it possible to filter by author and/or store? Is it possible in Superset to have some kind of a dropdown menus on a chart page to select author and/or store?

**Comment by michael-s-molina** [comment, marked as answer]
Hi @thaikoh. Yes, it's possible with [dashboard filters](https://docs.preset.io/docs/dashboard-filtering). You can create a [virtual dataset](https://docs.preset.io/docs/virtual-datasets) using SQL to join the tables and then create the author and store filters.
